### ISSUE TITLE
Invalid class package  with noclasspath option

### ISSUE THREAD

**Issue by tdurieux** [INRIA/spoon#329]
Hi,

When I spoon a class (see example below) which contains external classes (imported but not in the classpath) with the noclasspath option, the packages of the external classes are not correct. Indeed the packages of the external classes are the local package (here ```myPackage```) and not the package declared in the imports (here ```externalPackage```).

Original file:
```JAVA
package myPackage;

import externalPackage.ExternalClass;

public class NoClassPathTestV0 {
	public void test() {
		assertEquals(ExternalClass.class, null);
	}
}
```
Spoon output:
```JAVA
package myPackage;

public class NoClassPathTestV0 {
    public void test() {
        assertEquals(myPackage.ExternalClass.class, null);
    }
}
```

**Comment by tdurieux** [comment]
This issue is already fixed with Spoon 4.3.
But further investigation is required to check if this Spoon 4.3 feature works with static imports.

**Comment by tdurieux** [comment]
As expected the static imports are not supported by Spoon.

# Original class with static import:
```Java
package myPackage;

import static pack1.A.staticMethod;

public class AutoImportTestV0 {
	public void test() {
		String results = staticMethod("test", "test");
	}
}
```

# Output of Spoon AutoImport
```Java
package myPackage;


public class AutoImportTestV0 {
    public void test() {
        String results = staticMethod("test", "test");
    }
}
```
# Output of Spoon NoAutoImport
```Java
package myPackage;


public class AutoImportTestV0 {
    public void test() {
        java.lang.String results = staticMethod("test", "test");
    }
}
```
